navit/navit_shipped.xml

```diff
@@ -38,7 +38,7 @@
 	<navit center="11.5666 48.1333" zoom="256" tracking="1" orientation="-1" recent_dest="250" drag_bitmap="0">
 		<!-- Use one of gtk_drawing_area, qt_qpainter or sdl. 
 		     On windows systems, use win32 -->
-		<graphics type="gtk_drawing_area"/>
+		<graphics type="qt5" w="792" h="547"/>
```

Comment:
We should revert this change and leverage sed or xslt from ci/run_linux.sh instead